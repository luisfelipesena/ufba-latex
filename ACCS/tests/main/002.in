8000 75000000 75000000
