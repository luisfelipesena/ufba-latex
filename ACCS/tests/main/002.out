5625000000008000
